packages/@aws-cdk/aws-appsync/lib/resolver.ts

```diff
@@ -93,6 +108,11 @@ export class Resolver extends Construct {
       { functions: props.pipelineConfig.map((func) => func.functionId) }
       : undefined;
 
+    // If runtime is specified, code must also be
+    if (props.runtime && !props.code) {
```

Comment:
mapping templates and code should also be mutually exclusive. can we add a check here as well?